src/fracas/hover-provider.ts

```diff
@@ -17,15 +17,16 @@ export class FracasHoverProvider implements vscode.HoverProvider {
     ): Promise<vscode.Hover | null> {
         // search for comments preceding the definition for the symbol at the cursor
         const definitions = await findDefinition(document, position, token);
-        const comments = await mapAsync(definitions, async (definition) => {
+        const comments_raw = await mapAsync(definitions, async (definition) => {
```

Comment:
The coding convention for typescript is camel case `commentsRaw` for variables.

Reply:
fixed!